src/quo2/components/avatars/wallet_user_avatar/component_spec.cljs

```diff
@@ -0,0 +1,21 @@
+(ns quo2.components.avatars.wallet-user-avatar.component-spec
+  (:require [quo2.components.avatars.wallet-user-avatar.view :as wallet-user-avatar]
+            [test-helpers.component :as h]))
+
+(h/describe "wallet user avatar"
+  (h/describe "View internal"
+    (h/test "Renders with default values"
+      (h/render
+       [wallet-user-avatar/view {}])
+      (h/is-truthy (h/query-by-label-text :wallet-user-avatar)))
+    (h/test "Renders with given parameters"
```

Comment:
tbh I don't find this test description useful. in some sense it's the same as "Renders initials correctly" but less descriptive of what it is actually testing. 
I think we only really need to test how the configuration is changing the component. For this configuration is tricky as the color is testing styles which I don't think we should do tbh, and for size it's also styling.